server/sonar-server/src/test/java/org/sonar/server/computation/queue/ReportSubmitterTest.java

```diff
@@ -194,7 +203,7 @@ public void submit_a_report_on_new_project_with_global_scan_permission() {
   }
 
   @Test
-  public void submit_a_report_on_existing_project_with_global_scan_permission() {
+  public void submit_a_report_on_existing_project_with_scan_permission_on_organization() {
     userSession.setGlobalPermissions(SCAN_EXECUTION);
```

Comment:
The test implementation is indeed not correct. `UserSession#hasComponentPermission()` badly fallbacks on deprecated `hasPermission()` whereas `hasOrganizationPermission()` should be used. 
That will be fixed in a dedicated commit. It requires a change available in one of your PR.